pkg/api/bytes.go

```diff
@@ -37,13 +37,13 @@ func (s *Service) bytesUploadHandler(w http.ResponseWriter, r *http.Request) {
 	if err != nil {
 		logger.Debugf("bytes upload: get putter:%v", err)
 		logger.Error("bytes upload: putter")
-		jsonhttp.BadRequest(w, nil)
+		jsonhttp.BadRequest(w, fmt.Sprintf("new stamper putter: %v", err))
```

Comment:
The reason why this was not there in the first place was that we did not want to leak all these implementation details to the users. For eg, stamper putter or split write etc don't mean anything to the users. I am not sure if we want to expose these things.

Instead, we should put some messages here that don't leak the implementation details. This is the same reason why the error logs dont log the actual errors. As these errors are not meant for users. This is why they only show in debug logs.

Reply:
reverted